pkg/storage/tpr/install_types.go

```diff
@@ -45,20 +61,48 @@ func InstallTypes(cs clientset.Interface) error {
 		}
 
 		glog.Infof("Creating Third Party Resource Type: %s", tpr.Name)
-		if _, err := tprs.Create(&tpr); err != nil {
-			glog.Errorf("Failed to create Third Party Resource Type: %s (%s))", tpr.Name, err)
+		errChan = make(chan error, 1)
+		defer close(errChan)
+		errChans[tpr.Name] = errChan
```

Comment:
@arschles The only problem with single error channel is it would be harder for us the know which resource the error is coming from, thus harder for us to determine with resource to skip `Polling` for.